app/lib/service/services.dart

```diff
@@ -105,14 +107,14 @@ Future<void> withServices(FutureOr<void> Function() fn) async {
 }
 
 /// Run [fn] with services.
-Future<void> withFakeServices({
-  required FutureOr<void> Function() fn,
+Future<R> withFakeServices<R>({
+  required FutureOr<R> Function() fn,
   Configuration? configuration,
   MemDatastore? datastore,
   MemStorage? storage,
 }) async {
   if (Zone.current[_pubDevServicesInitializedKey] == true) {
-    return await fork(() async => await fn());
+    return await fork(() async => await fn()) as R;
```

Comment:
Can we make a utility function wrapping `fork`, called `saneFork` or maybe just `fork` and we place it somewhere we can import it from...

That way we won't need this cast every time we do something with `fork`.